data/net/minecraft/client/gui/screens/inventory/AbstractContainerScreen.mapping

```diff
@@ -72,8 +76,8 @@ CLASS net/minecraft/client/gui/screens/inventory/AbstractContainerScreen
 	METHOD renderBg (Lcom/mojang/blaze3d/vertex/PoseStack;FII)V
 		ARG 1 matrixStack
```

Comment:
``matrixStack`` -> ``poseStack``

Apply this change to any other instances you find.